scripts/pipelines/azure-devops/pipeline_generator.sh

```diff
@@ -143,6 +143,20 @@ function checkInstallations {
     fi
 }
 
+function ensurePathFormat {
+    currentDirectory=$(pwd)
+
+    # When necessary, converts a relative path into an absolute path, and a Windows-style path (e.g. "C:\Users" or C:/Users) into a 
+    # Unix-style path using forward slashes (e.g. "/c/Users").
+    localDirectory=${localDirectory//'\'/"/"}
+    cd "${localDirectory}"
+    [ $? != "0" ] && echo -e "${red}The local directory: '${localDirectory}' cannot be found, please check the path." && exit 1
```

Comment:
```suggestion
    cd "${localDirectory}" || (echo -e "${red}Error: Local directory '${localDirectory}' does not exist. Check provided path." && exit 1)
```

Haven't tested, but this looks cleaner.

Reply:
This is suggestion is not working (Parentheses create a subshell, therefore the "exit" is effectively ignored) but I have changed it slightly so that the behavior is the same as before. 
`cd "${localDirectory}" || { echo -e "${red}Error: Local directory '${localDirectory}' does not exist. Check provided path (missing quotes?)."; exit 1; }`